cypress.json

```diff
@@ -1,3 +1,7 @@
 {
-  "baseUrl": "http://localhost:5050"
+  "baseUrl": "http://localhost:3000",
```

Comment:
Manage this port from the environment file

Reply:
There is a ticket #32 already raised for this.